src/app/ApplicationTemplate.Shared.Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -262,18 +262,18 @@
 					<local:DataValidationView PropertyName="FavoriteDadNames"
 											  Model="{Binding Form}"
 											  Margin="0,4,0,0">
-						<!-- List -->
 						<Grid Background="{StaticResource SurfaceBrush}"
 							  CornerRadius="4">
-							<!--Remove this line from listView Properties to avoid crash on Windows App: ue:ListViewBaseMultipleSelectionBehavior.SelectedItems="{Binding Form.FavoriteDadNames, Mode=TwoWay}"
-								TODO : https://dev.azure.com/nventive/Practice%20committees/_workitems/edit/251910-->
-							<ListView ItemsSource="{Binding DadNames}"
-									  ItemContainerStyle="{StaticResource CheckedListViewItemStyle}"
-									  SelectionMode="Single"
-									  IsItemClickEnabled="True" />
+							<ItemsControl ItemsSource="{Binding DadNames}">
+								<ItemsControl.ItemTemplate>
+									<DataTemplate>
+										<CheckBox Content="{Binding .}" />
```

Comment:
Why `{Binding .}` is it the same thing as just `{Binding}`?

Reply:
yes, i'd remove it if it's the standard

Reply:
We usually don't use the dot. From what I've seen in our internal applications.